src/popup/i18n/locales/de.json5

```
@@ -0,0 +1,162 @@
+{
+  // Component App
+  app: {
+    // Error message used if an error is raised during initialization of background scripts. Prompt users to contact the feedback.
+    bgInitError: "Ein schwerwiegender Fehler ist aufgetreten: \"{msg}\". Falls dieser Fehler immer noch aufkommt nach einem Neustart ihres Browsers, kontaktieren Sie @:app.settings.contact.value",
+    // Page Settings
+    settings: {
+      // Prompt users to go to the previous page
+      back: "Zurück",
+      // Name of page where users can change settings
+      label: "Einstellungen",
+      // Prompt users to enable desktop notification
+      enableNtf: {
+        // Label for enableNtf
+        label: "Schalte Benachrichtigungen ein",
+        // Description for enableNtf
+        description: 'Benachrichtigt dich, falls ein (un)geplanter Livestream startet ("Livestream startet in 10 Minuten")'
+      },
+      // Prompt users to enable 30-hour clock
+      enable30Hours: {
+        // Label for enable30Hours
+        label: "Aktiviere 30 Stunden Uhr",
+        // Description for enable30Hours
+        description: "Der Tag startet um 06:00, erweitere 24:00 bis 29:59 um 0:00 AM - 5:59 AM des nächsten Tag zu repräsentieren"
+      },
+      // Prompt users to sync settings between different devices with the same browser account logged in
+      shouldSyncSettings: {
+        // Label for shouldSyncSettings
+        label: "Sync Einstellungen",
+        // Description for shouldSyncSettings
+        description: "Teile dieselben Einstellungen auf mehrere Geräte mit demselben Browser Konto"
+      },
+      // Prompt users to switch the UI language
+      language: {
+        // Label for language
+        label: "Language",
+        // Description for language
+        description: "Ändere die UI Sprache",
+        // Supported locales, where keys are type of i18n.LanguageCode and values are translated into the according locale
+        locales: {
+          en: "English",
+          ja: "日本語",
+          "zh-CN": "简体中文",
+		      de: "Deutsch"
```

Extra `\t`s are inserted here. I think we should remove them.

```suggestion
          de: "Deutsch"
```